migrations/20200517211842-initial-table-creation.js

```diff
@@ -0,0 +1,77 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /*
+      Add altering commands here.
+      Return a promise to correctly handle asynchronicity.
+
+      Example:
+      return queryInterface.createTable('users', { id: Sequelize.INTEGER });
+    */
+    await queryInterface.createTable('authors', {
+      id: { type: Sequelize.INTEGER, autoIncrement: true, primaryKey: true },
+      nameFirst: { type: Sequelize.STRING, allowNull: false },
+      nameLast: { type: Sequelize.STRING, allowNull: false },
+      createdAt: { type: Sequelize.DATE, allowNull: false, defaultValue: Sequelize.literal('CURRENT_TIMESTAMP') },
+      updatedAt: {
+        type: Sequelize.DATE,
+        allowNull: false,
+        defaultValue: Sequelize.literal('CURRENT_TIMESTAMP ON CURRENT_TIMESTAMP')
```

Comment:
You're missing the `UPDATE` to tell it to change the value when it is updated. This causes this migration to throw errors.

```Javascript
defaultValue: Sequelize.literal('CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP')
```